src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $color-bg-white;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  box-sizing: border-box;
+  position: relative;
+  padding: 16px;
+
+  &__image {
+    display: block;
+    background-image: url(../images/imac.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 20px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), using fixed width and height for elements is not recommended unless it is necessary. Consider using padding or max-width/max-height to make the design more responsive.